packages/kit/src/runtime/client/client.js

```diff
@@ -2086,6 +2086,10 @@ function _start_router() {
 
 		const submitter = /** @type {HTMLButtonElement | HTMLInputElement | null} */ (event.submitter);
 
+		const target = (submitter?.hasAttribute('formtarget') && submitter?.formTarget) || form.target;
```

Comment:
is `hasAttribute` a type guard?

Reply:
oh whoops, I was trying to mimic line 2094 but I see that's because it wanted to avoid passing an empty string into the URL constructor